### DISCUSSION TITLE
How to mint in exsitted network?

### DISCUSSION THREAD

**Discussion by kth-tw** [ConsenSys/quorum#1602] [Q&A] [answered]
I have a private QBFT quorum network. If I didn't set alloc in genesis file when create network, is any way to mint token?

**Comment by baptiste-b-pegasys** [comment, marked as answer]
Hi you can check block rewards.
You can set blockReward to one fixed account, amount will be added per each block validated.
https://docs.goquorum.consensys.net/configure-and-manage/configure/consensus-protocols/qbft#rewards
